consensus/enclave/impl/src/lib.rs

```diff
@@ -210,7 +200,7 @@ impl ConsensusEnclave for SgxConsensusEnclave {
 
         // This should never fail since we validated the fee map at the beginning of
         // this method.
```

Comment:
Is this comment still true? We don't call the is valid map method anymore.

Reply:
good catch, the comment is not needed.